src/less/tables.less

```diff
@@ -698,3 +698,59 @@
   }
 }
 
+@media (max-width: @global-screen-small-max) {
+
+  .ReactTable {
+    .rt-table {
+      overflow-x: scroll;
+      margin-left: 102px;
+      width: 80%;
+
+      .rt-tbody {
+        overflow: hidden !important;
+        width: 330px !important;
+      }
+
+      .rt-head {
+        width: 330px !important;
+      }
+
+      .rt-td:nth-child(1),
+      .rt-th:nth-child(1) {
+        z-index: 1;
+        position: absolute !important;
+        width: 100px !important;
+        left: 4px;
+        top: auto;
+        background-color: #fff;
+      }
+
+      .rt-tr {
+        &.-odd {
+          .rt-td:nth-child(1) {
+            background-color: #00000008 !important;
+          }
+        }
+      }
+
+      .rt-th {
+        span {
+          margin-left: 1px;
+        }
+      }
+
+      .rt-td,
+      .rt-th {
```

Comment:
Can we use some of our global spacing variables rather than new pixel values? If not, just let me know, but want to make sure we keep everything consistent in the UI.

Reply:
Will try to replace them with the closest match from the variable list. +1 @juanfabrega